docs/src/components/base/colors/colors.html

```diff
@@ -0,0 +1,8 @@
+{% for color in colors %}
+  <div class="swatch h20 mb8">
+    <div class="example swatch-color bg-{{ color.colorName }}"></div>
```

Comment:
I don't see that it makes much of a difference how we display the color rules in the PL as these swatches don't have a production equivalent. 

I use divs because I mostly don't care about the semantic value of the structure when I'm making the prototype — I assume that others have a stronger opinion and will modify when necessary. Divs are just a simple way of getting the material down on the page.